packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -883,62 +873,49 @@ function ResizableColumnsTable(props) {
 
   let layoutState = useTableColumnResizeState({
     // Matches the width of the table itself
-    tableWidth: 300,
+    tableWidth: 400,
```

Comment:
Can we keep this as 300 or at least something a little less than 400? Its a bit too wide on my phone
![Screenshot_20230323_141212_Chrome](https://user-images.githubusercontent.com/9661127/227363468-5ffc8ce3-fa1b-4231-962a-c21bba457c63.jpg)